src/initializers/datadog/index.ts

```diff
@@ -0,0 +1,19 @@
+import requireInjected from '../../require-injected';
+import { getLogger } from '../log4js';
+
+let tracer;
+export default config => {
+  if (process.env.DD_SERVICE && process.env.DD_ENV) {
+    tracer = requireInjected('dd-trace').init();
+    tracer.use('koa', {
+      blacklist: config?.datadog?.blacklistedPaths?.split(',') || ['/health']
```

Comment:
🔧 Initialise datadog.blacklistedPaths = ['/health'] in diamorphosis and then diamorphosis does the split.
```suggestion
      blacklist: config?.datadog?.blacklistedPaths
```
Maybe name it `dd.blacklistedPaths` to match DD_SERVICE and DD_ENV?
And in env: DD_BLACKLISTED_PATHS if it needs to change through an env?

Reply:
I don't want to use DD_* as this convention is used by the dd-tracer itself.